qlib/finco/workflow.py

```diff
@@ -15,6 +15,7 @@ class WorkflowContextManager:
 
     def __init__(self) -> None:
         self.context = {}
+        self.epoch = 0
```

Comment:
If there is a learning workflow outside, `WorkflowContextManager` is not necessarily aware of the learning process.